### PR TITLE
Compliance - Include pagination within system rules table

### PR DESCRIPTION
**Pull request by dLobatog**
Before:
Screenshot_20190425_115857

After:
Screenshot from 2019-05-24 14-07-45

### CODE REVIEW

File: packages/inventory-compliance/src/SystemRulesTable.js [hunk 1]
```diff
@@ -372,13 +372,16 @@ class SystemRulesTable extends React.Component {
...
-                    <Pagination
-                        numberOfItems={ rows.length / 2 }
-                        onPerPageSelect={ this.setPerPage }
-                        page={ page }
-                        onSetPage={ this.setPage }
-                        itemsPerPage={ itemsPerPage }
-                    />
+                    <TableToolbar isFooter className="ins-c-inventory__table--toolbar">
+                        <Pagination
+                            page={ page }
```
Comment (karelhala) [inline packages/inventory-compliance/src/SystemRulesTable.js]: Can you also add `dropDirection="up"` so the per page selector can be seen.

**Comment by dLobatog** [comment]
@karelhala This is my first release from the monorepo. Apparently `npm run release` would not work, because of this:

```
npm run release

> @redhat-cloud-services/frontend-components-root@0.0.1 release /home/dlobatog/workspace/insights/frontend-components
> semantic-release

[4:18:55 PM] [semantic-release] › ℹ  Running semantic-release version 15.13.14
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@khala/npm-release-monorepo"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@khala/commit-analyzer-wildcard"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@khala/wildcard-release-notes"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@khala/npm-release-monorepo"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@khala/npm-release-monorepo"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[4:18:55 PM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[4:18:55 PM] [semantic-release] › ⚠  This run was not triggered in a known CI environment, running in dry-run mode.
[4:18:55 PM] [semantic-release] › ⚠  Run automated release from branch master in dry-run mode
[4:18:57 PM] [semantic-release] › ✖  The command "git push --dry-run https://github.com/RedHatInsights/frontend-components.git HEAD:master" failed with the error message /usr/libexec/git-core/git-remote-https: symbol lookup error: /usr/libexec/git-core/git-remote-https: undefined symbol: curl_global_sslset
.
[4:18:57 PM] [semantic-release] › ⚠  Skip step "fail" of plugin "@semantic-release/github" in dry-run mode
[4:18:57 PM] [semantic-release] › ✖  EGITNOPERMISSION The push permission to the Git repository is required.
semantic-release cannot push the version tag to the branch master on remote Git repository with URL https://github.com/RedHatInsights/frontend-components.git.

Please refer to the authentication configuration documentation to configure the Git credentials on your CI environment and make sure the repositoryUrl is configured with a valid Git URL.

{ SemanticReleaseError: The push permission to the Git repository is required.
    at module.exports (/home/dlobatog/workspace/insights/frontend-components/node_modules/semantic-release/lib/get-error.js:6:10)
    at run (/home/dlobatog/workspace/insights/frontend-components/node_modules/semantic-release/index.js:79:11)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  name: 'SemanticReleaseError',
  code: 'EGITNOPERMISSION',
  details:
   '**semantic-release** cannot push the version tag to the branch `master` on remote Git repository with URL `https://github.com/RedHatInsights/frontend-components.git`.\n\nPlease refer to the [authentication configuration documentation](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/ci-configuration.md#authentication) to configure the Git credentials on your CI environment and make sure the [repositoryUrl](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#repositoryurl) is configured with a [valid Git URL](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols).',
  semanticRelease: true }npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @redhat-cloud-services/frontend-components-root@0.0.1 release: `semantic-release`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @redhat-cloud-services/frontend-components-root@0.0.1 release script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dlobatog/.npm/_logs/2019-05-26T14_18_57_598Z-debug.log

```

Am I supposed to update the version manually and then push it to npm directly?

**Comment by dLobatog** [comment]
Everytime I have to write monorepo it reminds me of

**Comment by karelhala** [comment]
@dLobatog yes, as you mentioned manual version increase and `npm publish` is the right way. Treat this repo as bunch of regular packages and because we don't have automaic releases enabled (to catch as many bugs as possible) you have to do all these things manually.

**Pull request merged**